end-to-end-test/mocks/auth/platform_dxtrade.ts

```diff
@@ -0,0 +1,31 @@
+import { Context } from '../../utils/mocks/mocks';
+
+export default function mock_platform_dxtrade(context: Context) {
+    if (
+        'platform' in context.request &&
+        context.request.platform === 'dxtrade' &&
+        context.request.trading_servers === 1
```

Comment:
We have ts error here 
![image](https://github.com/binary-com/deriv-app/assets/93518187/0d5db8bd-810e-4675-b5fc-4631e4ce8528)